lldb/source/Plugins/Language/Swift/SwiftOptional.cpp

```diff
@@ -44,24 +44,46 @@ static PointerOrSP
 ExtractSomeIfAny(ValueObject *optional,
                  bool synthetic_value = false) {
   if (!optional)
-    return nullptr;
+    return PointerOrSP("Can't get value from null optional");
 
   static ConstString g_Some("some");
   static ConstString g_None("none");
+  
+  const char *name = optional->GetName().GetCString();
+  if (!name)
+    name = "<unknown>";
 
   ValueObjectSP non_synth_valobj = optional->GetNonSyntheticValue();
-  if (!non_synth_valobj)
-    return nullptr;
+  if (!non_synth_valobj) {
+    std::string err_msg("Can't get the underlying value object for '");
+    err_msg.append(optional->GetName().GetCString());
```

Comment:
I think this should this use `name` from above?

Reply:
Oops.  I already merged this, I'll put up another PR to fix this.  This has no functional effect, the answer will be the same, however.